contracts/OwnedPermissionManager.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./PermissionInterface.sol";
+
+contract OwnedPermissionManager is PermissionInterface, Ownable {
+    
+    mapping(address => bool) publishers;
+    mapping(address => bool) reviewers;
```

Comment:
I know that the default visibility is private, but it is a good practice to specify it.